lib/schemas.js

```diff
@@ -27,11 +27,15 @@ const defaults = {
 }
 
 const optionsSchema = Joi.object({
-  userConfigPath: Joi.string(),
-  baseDir: Joi.string(),
-  onConfig: Joi.func(),
+  baseConfig: Joi.string(),
+  baseDir: Joi.string().allow(null),
   environment: Joi.object().default(defaults.environment),
-  shortstopHandlers: Joi.object().default(defaults.shortstopHandlers)
+  onConfig: Joi.func(),
+  shortstopHandlers: Joi.object().default(defaults.shortstopHandlers),
+  userConfigPath: Joi.alternatives(
+    Joi.string(),
```

Comment:
allow a string for backwards compatibility

Reply:
I wasn't aware of `.single()`...very interesting 🤔  let me try that out. Thanks for the suggestion!